HW6.py

```diff
@@ -0,0 +1,31 @@
+def merge_elems(*elems):
+    for elem in elems:
+        if isinstance(elem, str) or not hasattr(elem, '__iter__'):
+            yield elem
+        else:
+            yield from merge_elems(*elem)
+
+a = [1, 2, 3]
+b = 6
+c = 'zhaba'
+d = [[1, 2], [3, 4]]
+
+for _ in merge_elems(a, b, c, d):
```

Comment:
actual output: 1 2 3 6 zhaba 1 2 3 4 
expected: 1 2 3 6 z h a b a 1 2 3 4 
Strings should be processed as other iterable objects unless they have length=1